datasets/gooaq/README.md

```diff
@@ -0,0 +1,202 @@
+  - [Dataset Structure](#dataset-structure)
+    - [Data Instances](#data-instances)
+    - [Data Fields](#data-fields)
+    - [Data Splits](#data-splits)
+  - [Dataset Creation](#dataset-creation)
+    - [Curation Rationale](#curation-rationale)
+    - [Source Data](#source-data)
+      - [Initial Data Collection and Normalization](#initial-data-collection-and-normalization)
+      - [Who are the source language producers?](#who-are-the-source-language-producers)
+    - [Annotations](#annotations)
+      - [Annotation process](#annotation-process)
+      - [Who are the annotators?](#who-are-the-annotators)
+    - [Personal and Sensitive Information](#personal-and-sensitive-information)
+  - [Considerations for Using the Data](#considerations-for-using-the-data)
+    - [Social Impact of Dataset](#social-impact-of-dataset)
+    - [Discussion of Biases](#discussion-of-biases)
+    - [Other Known Limitations](#other-known-limitations)
+  - [Additional Information](#additional-information)
+    - [Dataset Curators](#dataset-curators)
+    - [Licensing Information](#licensing-information)
+    - [Citation Information](#citation-information)
+    - [Contributions](#contributions)
+
+## Dataset Description
+
+- **Homepage:** [GooAQ 🥑: Google Answers to Google Questions!](https://github.com/allenai/gooaq)
+- **Repository:** [GooAQ 🥑: Google Answers to Google Questions!](https://github.com/allenai/gooaq)
+- **Paper:** [GOOAQ: Open Question Answering with Diverse Answer Types](https://arxiv.org/abs/2104.08727)
+- **Point of Contact:** [Daniel Khashabi](danielk@allenai.org)
+
+### Dataset Summary
+
+GooAQ is a large-scale dataset with a variety of answer types. This dataset contains over
+5 million questions and 3 million answers collected from Google. GooAQ questions are collected
+semi-automatically from the Google search engine using its autocomplete feature. This results in
+naturalistic questions of practical interest that are nonetheless short and expressed using simple
+language. GooAQ answers are mined from Google's responses to our collected questions, specifically from
+the answer boxes in the search results. This yields a rich space of answer types, containing both
+textual answers (short and long) as well as more structured ones such as collections.
+
+### Supported Tasks and Leaderboards
+
+[More Information Needed]
+
+### Languages
+
+The dataset contains samples in English only.
+
+## Dataset Structure
+
+### Data Instances
+
+Each row of the data file should look like this:
+```json
+{
+  "id": 3339543,
+  "question": "what is the difference between collagen and whey protein?",
+  "short_answer": null,
+  "answer": "The main differences between the amino acid profiles of whey and collagen are that whey contains all 9 essential amino acids, while collagen only has 8. ... Collagen is a fibrous protein found in the skin, cartilage, and bones of animals whereas whey comes from milk.",
+  "answer_type": "feat_snip"
+}
+```
```

Comment:
Here we expect an actual example in python, so I suggest having `None` instead of nul

````suggestion
Each row of the data file should look like this:
```
{
  "id": 3339543,
  "question": "what is the difference between collagen and whey protein?",
  "short_answer": None,
  "answer": "The main differences between the amino acid profiles of whey and collagen are that whey contains all 9 essential amino acids, while collagen only has 8. ... Collagen is a fibrous protein found in the skin, cartilage, and bones of animals whereas whey comes from milk.",
  "answer_type": "feat_snip"
}
```
````

Reply:
Should I change it while yielding as well?

Reply:
Oh done that. For `short_answer` and `answer`